src/App.js

```diff
@@ -0,0 +1,11 @@
+import Greetings from './components/greetings';
+
+function App() {
+  return (
+    <div>
+      <Greetings />
+    </div>
+  );
```

Comment:
- [ ] It will be great to ensure that the app component has a route from the react-router that points to the greetings component. At the moment all you are doing is displaying the component without any route. This is a major requirement for your project to be approved.